ansible/roles/pre_commit/defaults/main.yaml

```diff
@@ -1 +1 @@
-clang_format_version: 15.0.4
+clang_format_version: 14.0.6
```

Comment:
Reverted due to https://github.com/ssciwr/clang-format-wheel/issues/56.
https://github.com/autowarefoundation/autoware/actions/runs/3601518900/jobs/6067419429#step:4:1168

```
fatal: [localhost]: FAILED! => {"changed": false, "cmd": ["/usr/bin/pip3", "install", "clang-format==15.0.4"], "msg": "stdout: Collecting clang-format==15.0.4\n  Downloading clang-format-15.0.4.tar.gz (9.6 kB)\n  Installing build dependencies: started\n  Installing build dependencies: finished with status 'done'\n  Getting requirements to build wheel: started\n  Getting requirements to build wheel: finished with status 'done'\n  Preparing metadata (pyproject.toml): started\n  Preparing metadata (pyproject.toml): finished with status 'done'\nBuilding wheels for collected packages: clang-format\n  Building wheel for clang-format (pyproject.toml): started\n  Building wheel for clang-format (pyproject.toml): finished with status 'error'\nFailed to build clang-format\n\n:stderr:   error: subprocess-exited-with-error\n  \n  × Building wheel for clang-format (pyproject.toml) did not run successfully.\n  │ exit code: 1\n  ╰─> [224 lines of output]\n      \n      \n      --------------------------------------------------------------------------------\n      -- Trying \"Ninja\" generator\n      --------------------------------\n      ---------------------------\n      ----------------------\n      -----------------\n      ------------\n      -------\n      --\n      Not searching for unused variables given on the command line.\n      -- The C compiler identification is GNU 11.3.0\n      -- Detecting C compiler ABI info\n      -- Detecting C compiler ABI info - done\n      -- Check for working C compiler: /usr/bin/cc - skipped\n      -- Detecting C compile features\n      -- Detecting C compile features - done\n      -- The CXX compiler identification is GNU 11.3.0\n      -- Detecting CXX compiler ABI info\n      -- Detecting CXX compiler ABI info - done\n      -- Check for working CXX compiler: /usr/bin/c++ - skipped\n      -- Detecting CXX compile features\n      -- Detecting CXX compile features - done\n      -- Configuring done\n      -- Generating done\n      -- Build files have been written to: /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_cmake_test_compile/build\n      --\n      -------\n      ------------\n      -----------------\n      ----------------------\n      ---------------------------\n      --------------------------------\n      -- Trying \"Ninja\" generator - success\n      --------------------------------------------------------------------------------\n      \n      Configuring Project\n        Working directory:\n          /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build\n        Command:\n          cmake /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd -G Ninja -DCMAKE_INSTALL_PREFIX:PATH=/tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-install -DPYTHON_VERSION_STRING:STRING=3.10.6 -DSKBUILD:INTERNAL=TRUE -DCMAKE_MODULE_PATH:PATH=/tmp/pip-build-env-5_y4824b/overlay/local/lib/python3.10/dist-packages/skbuild/resources/cmake -DPYTHON_EXECUTABLE:PATH=/usr/bin/python3 -DPYTHON_INCLUDE_DIR:PATH=/usr/include/python3.10 -DPYTHON_LIBRARY:PATH=/usr/lib/aarch64-linux-gnu/libpython3.10.so -DPython_EXECUTABLE:PATH=/usr/bin/python3 -DPython_ROOT_DIR:PATH=/usr -DPython_INCLUDE_DIR:PATH=/usr/include/python3.10 -DPython_FIND_REGISTRY:STRING=NEVER -DPython_NumPy_INCLUDE_DIRS:PATH=/usr/lib/python3/dist-packages/numpy/core/include -DPython3_EXECUTABLE:PATH=/usr/bin/python3 -DPython3_ROOT_DIR:PATH=/usr -DPython3_INCLUDE_DIR:PATH=/usr/include/python3.10 -DPython3_FIND_REGISTRY:STRING=NEVER -DPython3_NumPy_INCLUDE_DIRS:PATH=/usr/lib/python3/dist-packages/numpy/core/include -DCMAKE_MAKE_PROGRAM:FILEPATH=/tmp/pip-build-env-5_y4824b/overlay/local/lib/python3.10/dist-packages/ninja/data/bin/ninja -DCMAKE_BUILD_TYPE:STRING=Release\n      \n      -- The C compiler identification is GNU 11.3.0\n      -- The CXX compiler identification is GNU 11.3.0\n      -- Detecting C compiler ABI info\n      -- Detecting C compiler ABI info - done\n      -- Check for working C compiler: /usr/bin/cc - skipped\n      -- Detecting C compile features\n      -- Detecting C compile features - done\n      -- Detecting CXX compiler ABI info\n      -- Detecting CXX compiler ABI info - done\n      -- Check for working CXX compiler: /usr/bin/c++ - skipped\n      -- Detecting CXX compile features\n      -- Detecting CXX compile features - done\n      -- Configuring done\n      -- Generating done\n      CMake Warning:\n        Manually-specified variables were not used by the project:\n      \n          PYTHON_EXECUTABLE\n          PYTHON_INCLUDE_DIR\n          PYTHON_LIBRARY\n          PYTHON_VERSION_STRING\n          Python3_EXECUTABLE\n          Python3_FIND_REGISTRY\n          Python3_INCLUDE_DIR\n          Python3_NumPy_INCLUDE_DIRS\n          Python3_ROOT_DIR\n          Python_EXECUTABLE\n          Python_FIND_REGISTRY\n          Python_INCLUDE_DIR\n          Python_NumPy_INCLUDE_DIRS\n          Python_ROOT_DIR\n          SKBUILD\n      \n      \n      -- Build files have been written to: /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build\n      [1/10] Creating directories for 'build-clang-format'\n      [1/10] Performing download step (download, verify and extract) for 'build-clang-format'\n      -- Downloading...\n         dst='/tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/build-clang-format-prefix/src/llvm-project-15.0.4.src.tar.xz'\n         timeout='none'\n         inactivity timeout='none'\n      -- Using src='https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.4/llvm-project-15.0.4.src.tar.xz'\n      -- [download 0% complete]\n      -- [download 1% complete]\n      -- [download 2% complete]\n      -- [download 3% complete]\n      -- [download 4% complete]\n      -- [download 5% complete]\n      -- [download 6% complete]\n      -- [download 7% complete]\n      -- [download 8% complete]\n      -- [download 9% complete]\n      -- [download 10% complete]\n      -- [download 11% complete]\n      -- [download 12% complete]\n      -- [download 13% complete]\n      -- [download 14% complete]\n      -- [download 15% complete]\n      -- [download 16% complete]\n      -- [download 17% complete]\n      -- [download 18% complete]\n      -- [download 19% complete]\n      -- [download 20% complete]\n      -- [download 21% complete]\n      -- [download 22% complete]\n      -- [download 23% complete]\n      -- [download 24% complete]\n      -- [download 25% complete]\n      -- [download 26% complete]\n      -- [download 27% complete]\n      -- [download 28% complete]\n      -- [download 29% complete]\n      -- [download 30% complete]\n      -- [download 31% complete]\n      -- [download 32% complete]\n      -- [download 33% complete]\n      -- [download 34% complete]\n      -- [download 35% complete]\n      -- [download 36% complete]\n      -- [download 37% complete]\n      -- [download 38% complete]\n      -- [download 39% complete]\n      -- [download 40% complete]\n      -- [download 41% complete]\n      -- [download 42% complete]\n      -- [download 43% complete]\n      -- [download 44% complete]\n      -- [download 45% complete]\n      -- [download 46% complete]\n      -- [download 47% complete]\n      -- [download 48% complete]\n      -- [download 49% complete]\n      -- [download 50% complete]\n      -- [download 51% complete]\n      -- [download 52% complete]\n      -- [download 53% complete]\n      -- [download 54% complete]\n      -- [download 55% complete]\n      -- [download 56% complete]\n      -- [download 57% complete]\n      -- [download 58% complete]\n      -- [download 59% complete]\n      -- [download 60% complete]\n      -- [download 61% complete]\n      -- [download 62% complete]\n      -- [download 63% complete]\n      -- [download 64% complete]\n      -- [download 65% complete]\n      -- [download 66% complete]\n      -- [download 67% complete]\n      -- [download 68% complete]\n      -- [download 69% complete]\n      -- [download 70% complete]\n      -- [download 71% complete]\n      -- [download 72% complete]\n      -- [download 73% complete]\n      -- [download 74% complete]\n      -- [download 75% complete]\n      -- [download 76% complete]\n      -- [download 77% complete]\n      -- [download 78% complete]\n      -- [download 79% complete]\n      -- [download 80% complete]\n      -- [download 81% complete]\n      -- [download 82% complete]\n      -- [download 83% complete]\n      -- [download 84% complete]\n      -- [download 85% complete]\n      -- [download 86% complete]\n      -- [download 87% complete]\n      -- [download 88% complete]\n      -- [download 89% complete]\n      -- [download 91% complete]\n      -- [download 92% complete]\n      -- [download 93% complete]\n      -- [download 94% complete]\n      -- [download 95% complete]\n      -- [download 96% complete]\n      -- [download 97% complete]\n      -- [download 98% complete]\n      -- [download 99% complete]\n      -- [download 100% complete]\n      -- Downloading... done\n      -- extracting...\n           src='/tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/build-clang-format-prefix/src/llvm-project-15.0.4.src.tar.xz'\n           dst='/tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/llvm-project'\n      -- extracting... [tar xfz]\n      -- extracting... [analysis]\n      -- extracting... [rename]\n      -- extracting... [clean up]\n      -- extracting... done\n      [3/10] No update step for 'build-clang-format'\n      [4/10] No patch step for 'build-clang-format'\n      [4/10] Performing configure step for 'build-clang-format'\n      CMake Error: CMake was unable to find a build program corresponding to \"Ninja\".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.\n      CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage\n      CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage\n      CMake Error: CMAKE_ASM_COMPILER not set, after EnableLanguage\n      -- Configuring incomplete, errors occurred!\n      See also \"/tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/llvm/CMakeFiles/CMakeOutput.log\".\n      FAILED: build-clang-format-prefix/src/build-clang-format-stamp/build-clang-format-configure /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/build-clang-format-prefix/src/build-clang-format-stamp/build-clang-format-configure\n      cd /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/llvm && /usr/bin/cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLLVM_ENABLE_PROJECTS=clang -GNinja /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/llvm-project/llvm && /usr/bin/cmake -E touch /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build/build-clang-format-prefix/src/build-clang-format-stamp/build-clang-format-configure\n      ninja: build stopped: subcommand failed.\n      Traceback (most recent call last):\n        File \"/tmp/pip-build-env-5_y4824b/overlay/local/lib/python3.10/dist-packages/skbuild/setuptools_wrap.py\", line 640, in setup\n          cmkr.make(make_args, install_target=cmake_install_target, env=env)\n        File \"/tmp/pip-build-env-5_y4824b/overlay/local/lib/python3.10/dist-packages/skbuild/cmaker.py\", line 672, in make\n          self.make_impl(clargs=clargs, config=config, source_dir=source_dir, install_target=install_target, env=env)\n        File \"/tmp/pip-build-env-5_y4824b/overlay/local/lib/python3.10/dist-packages/skbuild/cmaker.py\", line 703, in make_impl\n          raise SKBuildError(\n      \n      An error occurred while building with CMake.\n        Command:\n          cmake --build . --target install --config Release --\n        Install target:\n          install\n        Source directory:\n          /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd\n        Working directory:\n          /tmp/pip-install-42n5ektm/clang-format_ce6eda801cb5478ea0a225209d63c7bd/_skbuild/linux-aarch64-3.10/cmake-build\n      Please check the install target is valid and see CMake's output for more information.\n      [end of output]\n  \n  note: This error originates from a subprocess, and is likely not a problem with pip.\n  ERROR: Failed building wheel for clang-format\nERROR: Could not build wheels for clang-format, which is required to install pyproject.toml-based projects\n"}
```